src/v2/cookbook/client-side-storage.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Client-Side Storage
+type: cookbook
+order: 10
+---
+
+## Base Example
+
+Client-side storage is an excellent way to quickly add performance gains to an application. By storing data on the browser itself, you can skip fetching information from the server every time the user needs it. While especially useful when offline, even online users will benefit from using data locally versus a remote server. Client-side storage can be done with [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies), [Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API) (technically "Web Storage"), [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API), and [WebSQL](https://www.w3.org/TR/webdatabase/) (a deprecated method that should not be used in new projects). 
+
+In this cookbook entry we'll focus on Local Storage, the simplest of the storage mechanisms. Local Storage uses a key/value system for storing data. It is limited to storing only simple values but complex data can be stored if you are willing to encode and decode the values with JSON. In general, Local Storage is appropriate for smaller sets of data you would want to persist, things like user preferences or form data. Larger data with more complex storage needs would be better stored typically in IndexedDB. 
+
+Let's begin with a simple form based example:
+
+``` html
+<div id="app">
+  My name is <input v-model="name">
+</div>
+```
+
+This example has one form field bound to a Vue value called `name`. Here's the JavaScript:
+
+``` js
+const app = new Vue({
+  el:'#app',
+  data:{
+    name:''
+  },
+  mounted() {
+    if(localStorage.name) this.name = localStorage.name;
+  },
+  watch:{
+    name(newName,oldName) {

```

Comment:
I wouldn't include the second param here if it's going to remain unused.

Reply:
Normally I'd agree - but do you think it's worthwhile keeping it there just as a reminder of the watch API in general?

Reply:
No, I don't think so. You can link to the api section of the docs for details like this along with an explanation for clarity. If you wouldn't write the function in this manner for this example, I'd remove it because it makes it more complex and thus, harder to understand at a glance, even if only slightly.

Reply:
Ok - fair enough. Removed (and the codepen was edited as well).